source/examples/context.cpp

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: 2020 The Khronos Group Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <CL/sycl.hpp>
+
+int main() {
+  sycl::context context;
+  auto devices = context.get_info<sycl::info::context::devices>();
+  for (auto device : devices) {
```

Comment:
I guess you consider a `device` is cheap enough to copy like a shared pointer and thus it does not require using a reference here? Which seems OK to me in a simple reference guide.

Reply:
Yeah that was the thinking. I can change it to a reference if you prefer that.